docs/robots.txt.template

```diff
@@ -0,0 +1,10 @@
+User-Agent: *
+Sitemap: http://docs.soliditylang.org/sitemap.xml
+
+Allow: /en/latest/
+Allow: /en/v0.7.6/
+Allow: /en/v{{ LATEST_VERSION }}/
```

Comment:
@chriseth @cameel This single line could have been very easily replaced by bumpversion as discussed #11322 instead of having extra scripts. I do not mind having extra scripts, just "defending" the proposal.

Reply:
Yeah, that's why I created #11322 originally. But then I noticed that we only have version here and in one other place in the main `CMakeFiles.txt`. If we get more then switching to `bumpversion` would totally make sense but as it is now, either approach works. The script wins for me simply because we already have it :)